rust/start.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+echo "health_check_port: 8084
+server_config:
+  processor_config:
+    type: emojicoin_processor
+  postgres_connection_string: $DATABASE_URL
+  indexer_grpc_data_service_address: $GRPC_DATA_SERVICE_URL
+  indexer_grpc_http2_ping_interval_in_secs: 60
+  indexer_grpc_http2_ping_timeout_in_secs: 10
+  auth_token: $GRPC_AUTH_TOKEN
+  starting_version: $STARTING_VERSION
+  number_concurrent_processing_tasks: 1
```

Comment:
About that- I'm actually not sure if this truly stops it from processing in parallel, because the insertion queries are chunked and collected with tokio across a range of txs, even if you only have a single processor thread.

Also probably something we can discuss on call, but I'm fairly sure this processor is thread-safe, since it has no concept of the database state at any point (it's almost entirely insertions)- the only exception being when it deletes stale data from the 1m periodic events table, but it does so in an atomic transaction, so I think it'd be fine.

Buuut I mean I could be wrong, and I actually am not sure if the low overall throughput of my test data would result in any errors.

Main reason I bring this up is because if we truly want it to be sequential I think we would need to get rid of the chunking logic

Reply:
Added the warning though